test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Xml.Tests.ps1

```diff
@@ -1,64 +1,64 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+
 Describe "ConvertTo-Xml DRT Unit Tests" -Tags "CI" {
     BeforeAll {
         class fruit {
             [string] $name;
         }
     }
 
-    $customPSObject = [pscustomobject]@{ "prop1" = "val1"; "prop2" = "val2" }
+    $customPSObject = [PSCustomObject]@{ "prop1" = "val1"; "prop2" = "val2" }
     $newLine = [System.Environment]::NewLine
```

Comment:
Please move these two lines to `BeforeAll`.

Reply:
Fixed, thank you @iSazonov